src/python/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM anapsix/alpine-java:8_server-jre
+
+RUN apk update && apk upgrade
+RUN apk add python3
+RUN apk add g++ python3-dev
+RUN apk add curl tar perl
+
+RUN cd /usr/local && curl -l https://www.exasol.com/support/secure/attachment/55207/EXASOL_JDBC-6.0.4.tar.gz  | tar zxf -
```

Comment:
Wouldn't it make sense to have version (6.0.4) as a variable?

Reply:
yes, but the URL not only include the version number but also an attachment ID so I should have that in a variable as well, at this point IMHO is easier to just leave the entire URL inlined